config/main.py

```diff
@@ -225,6 +234,9 @@ def load_mgmt_config(filename):
                 expose_value=False, prompt='Reload config from minigraph?')
 def load_minigraph():
     """Reconfigure based on minigraph."""
+    #Stop services before config push
```

Comment:
Also for `config reload`?

Reply:
Make sense.. Anytime we restart service for new config, its ideal to stop and then start. I'll address this.